opentreemap/stormwater/templates/stormwater/Bioswale_add.html

```diff
@@ -6,9 +6,17 @@
 
 {% block resource_add_details %}
 <div class="resource-question">
-  {# Translators: Bioswale.singular means Bioswale #}
-  {% trans "How large is the area that drains into the {Bioswale.singular}?"|interpolate_string:term as drainage_area %}
-  {% trans "Approximate size is fine. Do not include the area of the {Bioswale.singular} itself."|interpolate_string:term as explanation %}
+  {% captureas drainage_area %}
+    {% blocktrans with bioswale=term.Bioswale.singular %}
```

Comment:
As we discussed, I think we should stick with e.g. `Bioswale.terminology.singular` until we can make a wholesale switch to your proposed unification of singular/plural terms for both models and term substitutions.